docs/src/index.md

```diff
@@ -0,0 +1,87 @@
+# MLJFlux.jl
+
+A Julia package integrating deep learning Flux models with MLJ.
+
+## Objectives
+
+- Provide a user-friendly and high-level interface to fundamental [Flux](https://fluxml.ai/Flux.jl/stable/) deep learning models while still being extensible by supporting custom models written with Flux
+
+- Make building deep learning models more convenient to users already familiar with the MLJ workflow
+
+- Make it easier to apply machine learning techniques provided by MLJ, including: out-of-sample performance evaluation, hyper-parameter optimization, iteration control, and more, to deep learning models
+
+!!! note "MLJFlux Coverage"
+    MLJFlux support is focused on fundamental and widely used deep learning models; sophisticated architectures or techniques such as online learning, reinforcement learning, and adversarial networks are currently beyond its scope. 
+
+Also note that MLJFlux is limited to training models only when all training data fits into memory, though it still supports automatic batching of data.
+
+## Installation
+
+```julia
+import Pkg
+Pkg.activate("my_environment", shared=true)
+Pkg.add(["MLJ", "MLJFlux", "Flux"])
+```
+You only need `Flux` if you need to build a custom architecture or experiment with different optimizers, loss functions and activations.
+
+## Quick Start
+First load and instantiate mode:
+```@example
+using MLJ, Flux, MLJFlux
+import RDatasets
+
+# 1. Load Data
+iris = RDatasets.dataset("datasets", "iris");
+y, X = unpack(iris, ==(:Species), colname -> true, rng=123);
+
+# 2. Load and instantiate model
+NeuralNetworkClassifier = @load NeuralNetworkClassifier pkg="MLJFlux"
+clf = NeuralNetworkClassifier(
+    builder=MLJFlux.MLP(; hidden=(5,4), σ=Flux.relu),
+    optimiser=Flux.ADAM(0.01),
+    batch_size=8,
+    epochs=100, 
+    acceleration=CUDALibs()         # For GPU support
+    )
+
+# 3. Wrap it in a machine in fit
+mach = machine(clf, X, y)
+fit!(mach)
```

Comment:
We don't need to `fit!` if we're going to evaluate.
```suggestion
```

Reply:
Oops. Such a rookie mistake. Thanks. 

P.S.: Need to modify the comment as well.